### ISSUE TITLE
I want to automate capture of original css.

### ISSUE THREAD

**Issue by bokuweb** [bokuweb/re-bulma#80]

**Comment by kresli** [comment]
are you referring to src/styles? Are those styles duplication of bulma.css?

**Comment by bokuweb** [comment]
yes. I refer to` src / styles` . It is copied from bulma.css v0.2. we need to catch up latest bulma.css.Then automate it.

**Comment by kresli** [comment]
why not just use sass from bulma in node_modules?

**Comment by bokuweb** [comment]
I wanted to scope with minimal css referenced by component, but it failed now. If we have a better way, we may use sass.
